sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -3125,6 +3125,29 @@ public async Task SyncCopyFromUriAsync()
             Assert.AreEqual(CopyStatus.Success, copyResponse.Value.CopyStatus);
         }
 
+        [RecordedTest]
+        [ServiceVersion(Min = BlobClientOptions.ServiceVersion.V2020_12_06)]
+        public async Task SyncCopyFromUriAsync_EncryptionScope()
+        {
+            // Arrange
+            await using DisposingContainer test = await GetTestContainerAsync();
+            BlockBlobClient sourceBlob = InstrumentClient(test.Container.GetBlockBlobClient(GetNewBlobName()));
+            BlockBlobClient destBlob = InstrumentClient(test.Container.GetBlockBlobClient(GetNewBlobName()));
+            destBlob = InstrumentClient(destBlob.WithEncryptionScope(TestConfigDefault.EncryptionScope));
+
+            // Upload data to source blob
+            byte[] data = GetRandomBuffer(Constants.KB);
+            using Stream stream = new MemoryStream(data);
+
+            await sourceBlob.UploadAsync(stream);
+
+            // Act
+            Response<BlobCopyInfo> response = await destBlob.SyncCopyFromUriAsync(sourceBlob.Uri);
+
+            // Assert
+            //Assert.AreEqual(TestConfigDefault.EncryptionScope, response.Value.EncryptionScope);
```

Comment:
is there any other API that could return encryption scope? maybe get properties? would be good to have some assertion.

Reply:
This is an interesting point.  Blob Sync Copy now returned x-ms-encryption-scope as a response header.  I chose not to add it as a property to [BlobCopyInfo](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Azure.Storage.Blobs/src/Models/BlobCopyInfo.cs) because it is also the return type for Blob Async Copy, when does not accept Encryption Scope as a parameter, so it shouldn't return x-ms-encryption-scope in the response.

I'd be ok added Encryption Scope to BlobCopyInfo, but I think we should run it by the Arch Board first and see what they think.

Reply:
The BlobCopyInfo is a return value so I think it should be fine to add it - it's not something that user would get wrong (like incoherent inputs). The https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob returns plenty of headers that are optional depending on blob type and other things - that's a good precedent to follow.

Regardless of that, we shouldn't leave a test without assertion, so please find alternative way of asserting that blob got created with encryption scope until the decision is reached.

Reply:
Fixed.